### DISCUSSION TITLE
docs/micro-service/config

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#90] [General]
# docs/micro-service/config

服务端的配置方式，包括如何通过GoFrame框架读取和管理配置文件。提供了一份完整的配置模板示例，涵盖服务名称、服务监听地址、日志存储目录、错误日志记录及访问日志记录的设置方法。该配置与框架自动读取逻辑一致，确保了便捷的服务部署和高效的日志管理，以及如何设置和使用参数日志组件的配置进行独立的grpc server日志管理。

https://goframe.org/docs/micro-service/config

**Comment by korialuo** [comment]
请问config.yaml中，grpc下，可以配置多个服务吗？如：

**Reply by korialuo** [reply]
```
grpc:
   server1:
      name: "server1"
      address: ":8800"
   server2:
      name: "serrver2"
      address: "8801"

**Reply by gqcn** [reply]
> 请问config.yaml中，grpc下，可以配置多个服务吗？如：

No, it does not support multiple grpc server listening in configuration file.
